### ISSUE TITLE
Specific pages from businessinsider.com are displaying incorrect headlines in Reader View

### ISSUE THREAD

**Issue by avaida** [mozilla/readability#168]
**Affected**:
* 38.0b5-build1
* 39.0a2 (2015-04-16)
* 40.0a1 (2015-04-16)

**Sample pages**:
* http://www.businessinsider.com/global-market-update-april-17-2015-4
* http://www.businessinsider.com/citi-dollar-long-trade-is-taking-pause-2015-4

**Screenshot**:
![screenshot from 2015-04-17 17 19 41](https://cloud.githubusercontent.com/assets/5707237/7203788/fc12ca94-e525-11e4-951a-a1d595b6a4ce.png)

**Comment by n1k0** [comment]
Two side problems here:

First is that command `$ node test/generate-testcase.js bi-1 http://www.businessinsider.com/global-market-update-april-17-2015-4` retrieves an empty html document (Ajax contents retrieval issue I suspect).

Second is that even if we manually create the source.html file with appropriate content:

```
$ node test/generate-testcase.js bi-1
JSDOMParser error: expected '</div>'

JSDOMParser error: expected '</div>'

JSDOMParser error: expected '</div>'

JSDOMParser error: expected '</div>'

JSDOMParser error: expected '</div>'

JSDOMParser error: expected '</div>'

JSDOMParser error: expected '</div>'

JSDOMParser error: expected '</body>'

JSDOMParser error: expected '</html>'

[TypeError: Cannot read property 'firstElementChild' of undefined]

/Users/niko/Sites/readability/test/generate-testcase.js:112
    ex.stack.forEach(console.log.bind(console));
             ^
TypeError: Object TypeError: Cannot read property 'firstElementChild' of undefined
    at Object.Readability.parse (/Users/niko/Sites/readability/Readability.js:1693:41)
    at runReadability (/Users/niko/Sites/readability/test/generate-testcase.js:109:26)
    at /Users/niko/Sites/readability/test/generate-testcase.js:92:5
    at Object.oncomplete (fs.js:108:15) has no method 'forEach'
    at runReadability (/Users/niko/Sites/readability/test/generate-testcase.js:112:14)
    at /Users/niko/Sites/readability/test/generate-testcase.js:92:5
    at Object.oncomplete (fs.js:108:15)
```

I agree it would be nice to emulate/reproduce the preliminary sanitisation performed by FFx' XMLSerializer in our node test environment to avoid these issues, which really get in the way of headless investigations.

**Comment by n1k0** [comment]
Note as a side note that this is very issue is fixed using patch from #167

![screen shot 2015-04-17 at 17 48 38](https://cloud.githubusercontent.com/assets/41547/7205492/051ba792-e52a-11e4-9c4c-59fe867e5172.png)

**Comment by n1k0** [comment]
It's now landed, marking as resolved with #167.
